src/prefect/server/models/deployments.py

```diff
@@ -106,11 +106,6 @@ async def create_deployment(
     result = await session.execute(query)
     model = result.scalar()
 
-    if model.work_queue_name:
-        await models.work_queues._ensure_work_queue_exists(
-            session=session, name=model.work_queue_name, db=db
-        )
```

Comment:
I see that there's a call like this in `update_deployment` in this file as well which should also be removed. Would you be willing to remove it as part of this PR?

Reply:
Ah yes, I missed that one. Removed the call there as well!